Code/Source/Frame/ROS2Transform.cpp

```diff
@@ -81,6 +32,12 @@ namespace ROS2
 
     void ROS2Transform::Publish(const AZ::Transform& transform)
     {
-        m_transformPublisher->Publish(CreateTransformMessage(transform));
+        static bool isPublished = false;
```

Comment:
I think ```isPublished``` should be moved to a private member of ROS2Transfrom class instead of declaring it static in this function

Reply:
Especially if there can be multiple ROS2Transform instances.